homeassistant/components/heos/strings.json

```diff
@@ -91,8 +91,14 @@
     }
   },
   "exceptions": {
```

Comment:
Looks like you can also set [exception-translations](https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/exception-translations)

Reply:
There's at least 1 other place unrelated to this change that isn't using an exception translation. I'll get that next time. :)